alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/AbstractTikaMetadataExtractor.java

```diff
@@ -269,30 +269,30 @@ private ParseContext buildParseContext(Metadata metadata, String sourceMimeType)
 
             // Get the subject and description, despite things not
             //  being nearly as consistent as one might hope
```

Comment:
The deprecated OfficeOpenXMLCore.SUBJECT is now null upgrading from Tika 2.2.0 to 2.2.1